provider/all/packet.go

```diff
@@ -0,0 +1,11 @@
+// Copyright 2019 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// +build !minimal provider_vsphere
```

Comment:
The build tags need to be updated here.

Reply:
`provider_equinix` right?

Reply:
Actually I will leave packet for the time being until I do rename to everything.